config/estuary.go

```diff
@@ -101,6 +102,13 @@ func NewEstuary(appVersion string) *Estuary {
 			AggregateInterval:       time.Minute * 5,                                         // aggregate staging buckets every 5 minutes
 		},
 
+		Pinning: Pinning{
+			RetryWorker: RetryWorker{
+				Interval:   1,
```

Comment:
This should be of type `time.Duration` and set to `1*time.Hour` (more descriptive, tells us this is in hours)